corehq/apps/hqwebapp/static/registration/less/registration-main.less

```diff
@@ -13,6 +13,10 @@
   margin-top: 10vh;
   transition: margin-top .5s ease-in-out;
 
+  a.btn-primary {
```

Comment:
Is there a `btn-primary` that should just be switched from a link to a button? I think there are other specific styles we apply to `.btn-primary`, like bolding the text.

Reply:
hmmm? I don't follow. something in the styling is causing just the text to be blue.

Reply:
I mean, whatever link has blue text, could that link be changed to a button?

Reply:
it's a link tho...to a new page

Reply:
aha, got it, I had thought it was a pre-existing element